util/runtime.sh

```diff
@@ -145,6 +145,21 @@ if check_target "$TARGET_DIR"; then
     exit 0
 fi
 
+case $OSTYPE in
+    darwin*)
+        if [[ "$RUNTIME" = "wasmtime" ]]; then
+            # Mach integration in Wasmtime doesn't support forking and needs to
+            # be disabled in favor of posix signals.
+            #
+            # Patching is required until a new version of Wasmtime is released
+            # containing the commit `1c7a07b`.
+
+            notice "installing wasmtime on macOS, forcing 'build' mode"
+            MODE="build"
+        fi
+    ;;
+esac
```

Comment:
If we bump Wasmtime, this could be removed, right? What were the issues with bumping Wasmtime and should we merge a bump before merging this PR?

Reply:
Yeah, we can bump Wasmtime version before merging this PR. Bumping Wasmtime version implies updating a few backtrace patters in `t/01-wasm/directives/006-backtraces_directive.t` and some tests in `t/01-wasm/directives/008-flag_directive_wasmtime.t` to conform with new configuration rules adopted by Wasmtime in recent versions.